### DISCUSSION TITLE
v0.4.5-alpha

### DISCUSSION THREAD

**Discussion by bowring** [CIRDLES/Tripoli#209] [Show and tell]
To use OGTripoli mode, simply make a new analysis and load your file.
OGTripoli mode opens these files:
Phoenix ".TIMSDP"
Triton ".exp"
NU: (Individual Answers) ".txt"

[Check me out!](https://github.com/CIRDLES/Tripoli/blob/main/TripoliCore/src/main/resources/org/cirdles/tripoli/docs/ogTripoliHelp.md)

Please join the discussion.

## What's Changed
* Progress commit by @bowring in https://github.com/CIRDLES/Tripoli/pull/207
* OGTripoli plots for review by @bowring in https://github.com/CIRDLES/Tripoli/pull/208

**Full Changelog**: https://github.com/CIRDLES/Tripoli/compare/v0.4.4-alpha...v0.4.5-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/CIRDLES/Tripoli/releases/tag/v0.4.5-alpha'>v0.4.5-alpha</a>.</em>

**Comment by noahmclean** [comment]
- [x] To Do: Document ET_Redux-specific outputs as .xml for Pb and U analyses.
